### ISSUE TITLE
rule not drawn completely if used in combination with colspan

### ISSUE THREAD

**Issue by davewood** [PhilterPaper/PDF-Table#65]
```
      my $data = [
          [1,1],
          [2],
          [3,3],
      ];
      my $row_props = [
        {h_rule_w=>1},
        {h_rule_w=>1},
        {},
      ];
      my $cell_props = [
          [],
          [{colspan=>2}],
          [],
      ];  
      $table->table(
          $pdf, $page, $data,
          x => $x, y => $y, w => $w, h => $h,
          border_w => 0,
          cell_props => $cell_props,
          row_props => $row_props,
      );
```

draws this table

```
1     1
--------
2     
----
3     3
```

seems like the rule is not drawn over the whole length of the spanned cell.

**Comment by PhilterPaper** [comment]
It works for me. This problem (#63) was fixed in release 1.002 -- are you sure you're using the latest release?

**Comment by davewood** [comment]
my bad, i was still running 1.001

thanks
